### PR TITLE
Adding word 'Enterprise' where appropriate.

### PR DESCRIPTION
**Pull request by rdebusscher**
Other changes will be added in separate PRs

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/ecosystem/gradle-plugin.adoc [hunk 1]
```diff
@@ -80,15 +80,15 @@ ID and version. This configuration element has the higher precedence compared to
...
 
 |`daemon`
 |false
-|Starts payara-micro in separate JVM process and continues with the gradle build.
+|Starts Payara Micro in separate JVM process and continues with the gradle build.
 
 |`immediateExit`
 |false
-|If payara-micro is executed in daemon mode, the executor thread will wait for
+|If Payara Mmicro is executed in daemon mode, the executor thread will wait for
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/ecosystem/gradle-plugin.adoc:91]: Typo: Mmicro

File: docs/modules/ROOT/pages/documentation/ecosystem/security-connectors.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 = Security Connectors
 
-Security Connectors are a collection of Jakarta EE Security adaptor that implements
+Security Connectors are a collection of Jakarta EE Security adaptors that implements
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/ecosystem/security-connectors.adoc:3]: Isn't the word "adapters"?

File: docs/modules/ROOT/pages/documentation/payara-micro/starting-instance.adoc [hunk 3]
```diff
@@ -27,7 +27,7 @@ java -jar payara-micro.jar --outputlauncher --rootdir $rootDirectory
 ----
 
 This will create file `$rootDirectory/launch-micro.jar` and exit immediately.
-After that, it is possible to launch payara with plain
+After that, it is possible to launch Payara with plain
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/payara-micro/starting-instance.adoc:30]: Should be Payara Micro, in tandem with how the product is referred.

File: docs/modules/ROOT/pages/documentation/payara-server/public-api/twoidentitystores.adoc [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 _Since Payara Server 5.182_
 
-The Payara API provides a way to authenticate with two identity stores at the same time using the `@TwoIdentityStoreAuthenticationMechanismDefinition` annotation. By default a user is validated if they are authenticated by any identity store, this mechanism will only validate them if both stores authenticate it succesfully.
+The Payara API provides a way to authenticate with two identity stores at the same time using the `@TwoIdentityStoreAuthenticationMechanismDefinition` annotation. By default a user is validated if they are authenticated by any identity store, this mechanism will only validate them if both stores authenticate it succesfull.
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/payara-server/public-api/twoidentitystores.adoc:6]: Typo: "succesfull"

File: docs/modules/ROOT/pages/documentation/payara-server/server-configuration/jsf-options.adoc [hunk 5]
```diff
@@ -36,7 +36,7 @@ The `fish.payara.faces.enableParallelInit` context parameter can be combined wit
 </context-param>
 ----
 
-This is an existing (non Payara specific) parameter to also enable parallel initialisation, but in a slightly earlier stage of the initialisation process. As an payara specific implementation  this will use the Platform Concurrency executor if there is one, if not it will create a single use executor with 5 threads.
+This is an existing (non Payara specific) parameter to also enable parallel initialisation, but in a slightly earlier stage of the initialisation process. As an Payara specific implementation  this will use the Platform Concurrency executor if there is one, if not it will create a single use executor with 5 threads.
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/payara-server/server-configuration/jsf-options.adoc:39]: should say: "Payara Platform"?

File: docs/modules/ROOT/pages/documentation/payara-server/public-api/twoidentitystores.adoc (hunk 4 above)
Comment (rdebusscher) [reply docs/modules/ROOT/pages/documentation/payara-server/public-api/twoidentitystores.adoc]: successfully seems after all grammatically more correct (but needs double s)

**Pull request merged**